tests/tests_core/test_experiment.py

```diff
@@ -241,6 +241,47 @@ def test_bayes_precision_delta_derived_kpis(self):
 
         # self.getExperiment([self.derived_kpi_1['name']], [self.derived_kpi_1]).delta()
 
+    def test_quantile_filtering(self):
+        exp = self.getExperiment([self.derived_kpi_1['name']], [self.derived_kpi_1])
+        exp.filter(
+            kpis=[
+                'normal_same',
+                'normal_shifted',
+                'normal_shifted_by_feature',
+                'normal_unequal_variance'
+            ]
+        )
+        self.assertEqual(len(self.data) - len(exp.data), exp.metadata['filtered_entities_number'])
+
+    def test_quantile_filtering_2(self):
```

Comment:
I would give it a more meaningful name. Also for the other quantile filtering tests.